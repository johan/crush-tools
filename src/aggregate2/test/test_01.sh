test_number=01
description="multi-key, multi-sum"

infile=$test_dir/test.in
outfile=$test_dir/test_$test_number.out
expected=$test_dir/test_$test_number.expected

$bin -p -k 1,2 -s 3,4 $infile > $outfile

if [ $? -ne 0 ] ||
   [ "`diff -q $outfile $expected`" ]; then
  test_status $test_number 1 "$description" FAIL
else
  test_status $test_number 1 "$description" PASS
  rm "$outfile"
fi
