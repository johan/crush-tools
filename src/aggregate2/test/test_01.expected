Text-1	Text-2	Numeric-1	Numeric-2
first text value	a	2	13
first text value	b	1	3
second text value	b	2	2
second text value	c	4	4
second text value	d	2	7
